translate.py

```diff
@@ -1,11 +1,24 @@
 import re
 from pathlib import Path
-
+import numpy as np
```

Comment:
Why numpy, not builtin random?
If numpy, then add requirement in requirements.txt.